atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -462,13 +464,19 @@ def run(self):
         """
         Run the plugin.
         """
+
+        # get the session and token information in case we need to refund a failed build
+        self.session = get_koji_session(self.workflow, self.koji_fallback)
+        build_token = self.workflow.reserved_token
+        build_id = self.workflow.reserved_build_id
+
         # Only run if the build was successful
         if self.workflow.build_process_failed:
             self.log.info("Not importing failed build to koji")
+            if self.reserve_build and build_token is not None:
+                build_info = self.session.CGRefundBuild(PROG, build_id, build_token)
```

Comment:
The call returns `None` on success, and anyway we don't use the return value, so we should just remove `build_info = ` at the start of this line.